### PR TITLE
[CLI] Generate Example `config.yml` anywhere via `llmtune generate config`

### PR DESCRIPTION
**Pull request by benjaminye**
Addresses https://github.com/georgian-io/LLM-Finetuning-Toolkit/issues/129

- This PR introduces `llmtune generate config` commands
- Since this package now installs via `pip`/`pipx`, users are no longer directly exposed to the entire repo
- This necessitates commands to generate `config.yml` on the fly for a better user experience (as opposed to having to copy from this repo directly)

### CODE REVIEW

**Review by RohitSaha — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,11 +38,14 @@ This guide contains 3 stages that will enable you to get the most out of this to
...
-   llmtune --config-path ./config.yml
+```shell
+   llmtune generate config
+   llmtune run --config-path ./config.yml
 ```
 
-This command initiates the fine-tuning process using the settings specified in the default YAML configuration file `config.yaml`.
+The first command generates an example `config.yml` file and saves in the current working directory.
+
+Then the second command initiates the fine-tuning process using the settings specified in the default YAML configuration file `config.yaml`.
```
Comment (truskovskiyk) [inline README.md:48]: could you add an explanation as to why we need to generate config.yaml?

File: llmtune/cli/toolkit.py [hunk 2]
```diff
@@ -1,13 +1,17 @@
 import logging
-from os import listdir
-from os.path import exists, join
+import shutil
+from os import getcwd, listdir
+from os.path import dirname, exists, join
```
Comment (truskovskiyk) [inline llmtune/cli/toolkit.py:4]: please use https://docs.python.org/3/library/pathlib.html, it's a much better library then is.path

File: README.md (hunk 1 above)
Comment (benjaminye) [reply README.md]: resolved in [e5c9c73](https://github.com/georgian-io/LLM-Finetuning-Toolkit/pull/137/commits/e5c9c730ee8b547f4997e4c3ee32b9176cb35fa1)

File: llmtune/cli/toolkit.py (hunk 2 above)
Comment (benjaminye) [reply llmtune/cli/toolkit.py]: Agreed, much better. Addressed [9ef89d9](https://github.com/georgian-io/LLM-Finetuning-Toolkit/pull/137/commits/9ef89d9ebc36c1a0c695ac1550fbe40ca9e34f2f)...[0ac2a1d](https://github.com/georgian-io/LLM-Finetuning-Toolkit/pull/137/commits/0ac2a1db7bac1451454f8a4309588f10425f73b0)

**Pull request merged**